### ISSUE TITLE
Image appears as pixelated in this .pptx file

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/DesktopEditors#284]
**Do you want to request a *feature* or report a *bug*?**
Bug.
**What is the current behavior?**
Image appears pixelated in OnlyOffice.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Step: just open the .pptx file in OnlyOffice.
**What is the expected behavior?**
Left image should not appear pixelated.
**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
Linux version of DesktopEditors.

[TD08-2019.pptx](https://github.com/ONLYOFFICE/DesktopEditors/files/3902148/TD08-2019.pptx)

**Comment by Rita-Bubnova** [comment]
Hello, @parrenin.
Thank you. I can confirm - this is a bug, issue 43684 in our internal issue tracker.

**Comment by parrenin** [comment]
Thanks for the reply.
I am new to OnlyOffice and was wondering: is your internal issue tracker available to outside people?
Is it not too complicated to have two issue systems in parallel?

**Comment by Rita-Bubnova** [comment]
@parrenin, the issues with private user files are reported in our private tracker, so outside users don't have access to it.

**Comment by ShockwaveNN** [comment]
This issue should be fixed in ONLYOFFICE DesktopEditors v.5.5.1

Feel free to comment or open new issue if still have some troubles.

**Comment by parrenin** [comment]
I tried with 5.5.1. It is better than before, but still not as good as in LibreOffice on Linux.
I attach a screenshot comparing the rendering in both softwares.
![Capture d’écran de 2020-05-14 12-17-22](https://user-images.githubusercontent.com/4678272/81922826-f7259300-95dc-11ea-85df-a8377cabd85c.png)

**Comment by Rita-Bubnova** [comment]
@parrenin, unfortunately, in this file we can’t get a better raster image.
